docs/configuration/actions.md

```diff
@@ -268,3 +268,13 @@ actions:
 
 Please note that `%changelog` expects dashes in a changelog entry, but the dash
 would be interpreted by `echo` as an option so we need to quote it.
+
+-----
+If you want to execute something in a different directory then you need to specify 
+all the commands inside the same directive:
```

Comment:
```suggestion
Commands default to upstream git repo as a working directory. If you want to change it (to a dist-git repo for example), spawn a shell and change the directory as the first command:
```

Reply:
I also wanted to mention that upstream is the default execution dir but this is not always true. So I tried to be more generic, since this paragraph is not strictly related with propose_downstream and pull_from_upstream.

Reply:
Ok, then something like this:

```suggestion
If you want to change the working directory of a command or multiple commands, don't do this, it won't work as expected:

```yaml
actions:
  post-upstream-clone:
    - cd /tmp
    - touch test_file"
\```

Instead, spawn a shell and change the directory as the first command:
```